### PR TITLE
godep: Remove useless dep

### PR DESCRIPTION
**Pull request by gaocegege**
Signed-off-by: Ce Gao <gaoce@caicloud.io>

**What this PR does / why we need it**:

I imported a never used dep in lock, this PR is to remove it. Sorry about it.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/521)

### CODE REVIEW

File: Gopkg.lock [hunk 1]
```diff
@@ -33,14 +33,6 @@
   pruneopts = "T"
   revision = "de5bf2ad457846296e2031421a34e2568e304e35"
 
-[[projects]]
-  digest = "1:9187ad53cd8b43cc82d1615ca6319503735fd87cd2b4dbeb80c57831e9affd79"
-  name = "github.com/caicloud/serving-controller"
-  packages = ["pkg/consts"]
-  pruneopts = "T"
-  revision = "bec825cac979e73109541ca7518dd6b96ce9e68a"
-  version = "v0.0.1-rc.5"
```
Comment (hougangliu) [inline Gopkg.lock]: LGTM, but your PR just reminds me that maybe we should includes Gopkg* into prow, for example we upgrades some reps

Comment (gaocegege) [reply Gopkg.lock]: Actually, I am wondering why it passes the CI test. It should raise an error during dep ensure.

Comment (johnugeorge) [reply Gopkg.lock]: i did this in  https://github.com/kubeflow/katib/pull/512/files . It was failing for me

Comment (gaocegege) [reply Gopkg.lock]: Yeah, it should fail in CI, too. I am not sure why it passed.

Comment (gaocegege) [reply Gopkg.lock]: Anyway, I will add gopkg in prow config to trigger CI in this PR.

Comment (gaocegege) [reply Gopkg.lock]: Addressed

**Comment by gaocegege** [comment]
PTAL @hougangliu

**Comment by johnugeorge** [comment]
Do you have to merge this? I have to rebase my PR again  :)

**Comment by gaocegege** [comment]
@johnugeorge Oh Since you have changed it, I removed the changes about the gopkg.lock.

PTAL

**Comment by johnugeorge** [comment]
It didn't run tests as prow config was outdated. In general, i think that listing out directories is error prone in Katib as version changes spread across  the repo

**Comment by gaocegege** [comment]
Yeah, now we cannot find a better way to solve the problem. If we trigger v1alpha1 and v1alpha2 workflow for all PRs, it is easier to get CI errors. Let's keep it and see if there is a better solution.

File: prow_config.yaml [hunk 2]
```diff
@@ -38,6 +38,7 @@ workflows:
     - test/workflows/*
     - manifests/v1alpha1/*
     - scripts/v1alpha1/*
+    - Gopkg*
```
Comment (hougangliu) [inline prow_config.yaml]: can you also add it into postsubmit? since the new images maybe use new dep, which need be published to public registry

Comment (gaocegege) [reply prow_config.yaml]: Make sense.

Comment (gaocegege) [reply prow_config.yaml]: Addressed, PTAL

**Comment by hougangliu** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**